cmd/ocm/create/cluster/cmd.go

```diff
@@ -1727,6 +1748,20 @@ func promptGcpCustomEncryption(fs *pflag.FlagSet, connection *sdk.Connection) er
 	return nil
 }
 
+func promptEtcdEncryption(fs *pflag.FlagSet) error {
+	if !args.fipsEncryption {
+		return arguments.PromptBool(fs, "etcd-encryption")
+	}
+
+	if fs.Changed("etcd-encryption") && !args.etcdEncryption {
+		return fmt.Errorf("FIPS mode cannot have etcd encryption disabled")
```

Comment:
pls chane this to -  "When FIPS mode is enabled, etcd encryption cannot be disabled"